pkg/customresource/utils.go

```diff
@@ -14,73 +14,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package customresource
 
 import (
-	"fmt"
-	"runtime"
 	"strings"
 
-	"github.com/prometheus/common/version"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/dynamic"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/klog/v2"
 	testUnstructuredMock "k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1"
-
-	"k8s.io/kube-state-metrics/v2/pkg/customresource"
 )
 
 var config *rest.Config
 var currentKubeClient clientset.Interface
 var currentDiscoveryClient *discovery.DiscoveryClient
 var currentDynamicClient *dynamic.DynamicClient
 
-// CreateKubeClient creates a Kubernetes clientset and a custom resource clientset.
-func CreateKubeClient(apiserver string, kubeconfig string) (clientset.Interface, error) {
```

Comment:
I'm not sure why this was moved out?

Reply:
Remove this dependency
v2/pkg/util
└──v2/pkg/customresource

https://github.com/kubernetes/kube-state-metrics/issues/2103